config/database.yml

```diff
@@ -19,18 +19,18 @@ default: &default
   encoding: unicode
   # For details on connection pooling, see Rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  pool: <%= ENV.fetch("DATABASE_POOL_SIZE") { ENV.fetch("RAILS_MAX_THREADS", 5).to_i } %>
```

Comment:
shouldn't `DATABASE_POOL_SIZE` be present in the `.env_sample` as well? If we decide to keep the separate env var we might also need to explain that it should generally have the same value as `RAILS_MAX_THREADS` (which is why Rails by default doesn't introduce a separate env var). See https://github.com/rails/rails/pull/23528 as to why.

@mstruve @maestromac are we using at all `DATABASE_POOL_SIZE`? Note that's not present in the default Rails DB configuration https://github.com/rails/rails/blob/6-1-stable/railties/lib/rails/generators/rails/app/templates/config/databases/postgresql.yml.tt

Reply:
It's a great point. If possible I think it'd be simpler to drop DATABASE_POOL_SIZE from the .env_sample. 
Looking through the Rails source for `6.2.0` DATABASE_POOL_SIZE isn't used. 

I'll add `WEB_CONCURRENCY` to the env_sample as well.

Reply:
It could be worth noting the documention from Heroku too

> As you add more concurrency to your application, it will need more connections to your database. A good formula for determining the number of connections each application will require is to multiply the RAILS_MAX_THREADS by the WEB_CONCURRENCY. This combination will determine the number of connections each dyno will consume. 

https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#database-connections

Reply:
I agree that we should drop `DATABASE_POOL_SIZE`. Let's stay as close as possible to Rails and Puma defaults.

BTW it's also possible to change the pool size by changing the URL:

```shell
> DATABASE_URL=postgres://localhost/PracticalDeveloper_development?pool=10  rails c
[1] pry(main)> ActiveRecord::Base.connection_pool.stat
=> {:size=>10,
 :connections=>0,
 :busy=>0,
 :dead=>0,
 :idle=>0,
 :waiting=>0,
 :checkout_timeout=>2.0}
```

:)

Reply:
@rhymes sorry I missed this. yes we are using `DATABASE_POOL_SIZE` on production.

Reply:
ok cool, let's keep it then :)